src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -130,45 +145,59 @@ const StreamGroup = createReactClass({
     const {organization, query, selection} = this.props;
     const {data} = this.state;
 
-    // TODO: @taylangocmen when there is no discover query open events page
-
-    const {period, start, end} = selection.datetime || {};
+    // when there is no discover feature open events page
+    const hasDiscoverQuery = organization.features.includes('discover-basic');
 
-    const discoveryQueryTerms = [];
+    const queryTerms = [];
 
     if (filtered && query) {
       const queryObj = queryToObj(query);
       for (const queryTag in queryObj)
-        if (!['is', '__text'].includes(queryTag)) {
+        if (!DiscoveryExclusionFields.includes(queryTag)) {
           const queryVal = queryObj[queryTag].includes(' ')
             ? `"${queryObj[queryTag]}"`
             : queryObj[queryTag];
-          discoveryQueryTerms.push(`${queryTag}:${queryVal}`);
+          queryTerms.push(`${queryTag}:${queryVal}`);
         }
+
+      if (!hasDiscoverQuery && queryObj.__text) queryTerms.push(queryObj.__text);
     }
 
-    const additionalQuery =
-      (discoveryQueryTerms.length ? ' ' : '') + discoveryQueryTerms.join(' ');
-
-    const discoverQuery = {
-      id: undefined,
-      name: data.title || data.type,
-      fields: ['title', 'release', 'environment', 'user', 'timestamp'],
-      orderby: '-timestamp',
-      query: `issue.id:${data.id}${additionalQuery}`,
-      projects: [data.project.id],
-      version: 2,
-    };
+    const commonQuery = {projects: [data.project.id]};
+
+    const searchQuery = (queryTerms.length ? ' ' : '') + queryTerms.join(' ');
 
-    if (!!start && !!end) {
-      discoverQuery.start = start;
-      discoverQuery.end = end;
+    if (hasDiscoverQuery) {
+      const {period, start, end} = selection.datetime || {};
+
+      const discoverQuery = {
+        ...commonQuery,
+        id: undefined,
+        name: data.title || data.type,
+        fields: ['title', 'release', 'environment', 'user', 'timestamp'],
+        orderby: '-timestamp',
+        query: `issue.id:${data.id}${searchQuery}`,
+        version: 2,
+      };
+
+      if (!!start && !!end) {
+        discoverQuery.start = start;
+        discoverQuery.end = end;
+      } else {
+        discoverQuery.range = period || DEFAULT_STATS_PERIOD;
+      }
+
+      const discoverView = EventView.fromSavedQuery(discoverQuery);
+      return discoverView.getResultsViewUrlTarget(organization.slug);
     } else {
```

Comment:
could probably drop this else now that they both include returns

Reply:
Yeah I'll do that